pages/content/pixi/info-texts/field-and-lab-tests.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
Let's consolidate these texts into the core-web-vitals.md and use that, since the tooltip is next to the Core Web Vitals headline, correct?